src/common/components/traffic-lights/traffic-light.css

```diff
@@ -8,15 +8,14 @@ $signal-colour-default: $mid-grey;
 
 .signal {
   background: $signal-colour-default;
-  border-radius: 50%;
+  border-radius: 16px;
```

Comment:
I think it was just following spec: https://github.com/canonical-ols/build.snapcraft.io/issues/354#issuecomment-285355591

But yes, 50% would just work fine (as it's 50% of 32px)